publications/tasks.py

```diff
@@ -104,3 +110,38 @@ def harvest_oai_endpoint(url):
             parse_oai_xml_and_save_publications(response.content)
     except requests.exceptions.RequestException as e:
         print ("The requested URL is invalid or has bad connection.Please change the URL")
+
+def send_monthly_email(sent_by=None):
+    recipients = User.objects.values_list('email', flat=True)
+    last_month = now().replace(day=1) - timedelta(days=1)  # Get last month's last day
+    new_manuscripts = Publication.objects.filter(creationDate__month=last_month.month)
+
+    if not new_manuscripts.exists():
+        print("No new manuscripts found for this month. Skipping email.")
+        return
+
+    subject = "New Manuscripts This Month"
+    content = "Here are the new manuscripts:\n" + "\n".join([pub.title for pub in new_manuscripts])
+
+    for recipient in recipients:
+        print(f"Sending email to {recipient}")
+        try:
+            send_mail(
+                subject,
+                content,
+                settings.EMAIL_HOST_USER,
+                [recipient],
+                fail_silently=False,
+            )
+            SentEmailLog.log_email(recipient, subject, content, sent_by=sent_by)
+            print(f"Email sent successfully to {recipient}")
+        except Exception as e:
+            print(f"Failed to send email to {recipient}: {e}")
+
+def schedule_monthly_email_task():
+    schedule(
+        'publications.tasks.send_monthly_email',
+        schedule_type='MONTHLY',
+        repeats=-1,
+        next_run=datetime.now().replace(day=28, hour=23, minute=59)
```

Comment:
Is `next_run` here setting up the first execution? I'm a bit worried about the `28` in there... does that work for all months?